### DISCUSSION TITLE
javascipt is work or not in wiki.js

### DISCUSSION THREAD

**Discussion by nihalgurjar47** [requarks/wiki#6496] [Help / Questions]
How to enter Javascript into a wiki page?

When we add some HTML content in page and want some event(like button click call javascript function) is that is possible in wiki.js or not.

**Comment by NGPixel** [comment]
You need to insert your JS from the Page Properties dialog, in the Scripts tab.

**Comment by benborges** [comment]
Can it load a JS file from a subdomain of the same domain hosting the wiki ? 

or is it better to write down the entire content of the file into the wikijs script placeholder ?
